### PR TITLE
Profile Picture

### PR DESCRIPTION
**Pull request by Deepakyadav19**
profile page contains for html and css pages.
All nav-links are on working state.

### CODE REVIEW

File: Deepak_Yadav/P1 Profile/css/style.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+.background-front{
+    background: url(../images/background-front.jpg) no-repeat;
+    background-size: cover;
+    -webkit-filter: blur(0.5px);
+    min-height: 630px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
```
Comment (nikhilrstg18) [inline Deepak_Yadav/P1 Profile/css/style.css]: use flex-flow instead, its short hand property for flex-direction and flex-wrap

File: Deepak_Yadav/P1 Profile/html/about.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+        </div>
+    </nav>
+</header>
+
+<body>
+
+
+    <section>
+
```
Comment (nikhilrstg18) [inline Deepak_Yadav/P1 Profile/html/about.html]: get rid of extra line spaces
missing comments in layout

File: Deepak_Yadav/P1 Profile/css/style.css (hunk 1 above)
Comment (Deepakyadav19) [reply Deepak_Yadav/P1 Profile/css/style.css]: ohkk nikhil..i am updating this right now..

File: Deepak_Yadav/tic tac toe/index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="css/animate.css">
+    <title>Document</title>
+    
+    <style>
+        body{
```
Comment (nikhilrstg18) [inline Deepak_Yadav/tic tac toe/index.html]: this should belong in external CSS file

File: Deepak_Yadav/tic tac toe/scripts.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+function highLight(cell1, cell2, cell3, symbol){
+    btnDisable();
+    document.getElementById(cell1).style.backgroundColor = 'red';
+    document.getElementById(cell2).style.backgroundColor = 'red';
+    document.getElementById(cell3).style.backgroundColor = 'red';
+}
+
+//game will start from starting with a new different background color
+function reset(){
+    var b1 = document.getElementById("b1");
```
Comment (nikhilrstg18) [inline Deepak_Yadav/tic tac toe/scripts.js]: missing DRY KISS, extra code
code optimization required

File: Deepak_Yadav/tic tac toe/scripts.js [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+    var b3 = document.getElementById("b3").innerHTML;
+    var b4 = document.getElementById("b4").innerHTML;
+    var b5 = document.getElementById("b5").innerHTML;
+    var b6 = document.getElementById("b6").innerHTML;
+    var b7 = document.getElementById("b7").innerHTML;
+    var b8 = document.getElementById("b8").innerHTML;
+    var b9 = document.getElementById("b9").innerHTML;
+    
+
+    if(((b1=="X") || (b1=="O")) && ((b1 == b2) && (b2 == b3))){
```
Comment (nikhilrstg18) [inline Deepak_Yadav/tic tac toe/scripts.js]: missing DRY

File: Deepak_Yadav/tic tac toe/index.html (hunk 3 above)
Comment (Deepakyadav19) [reply Deepak_Yadav/tic tac toe/index.html]: ohkk nikhil..i will update this in a while.

File: Deepak_Yadav/tic tac toe/scripts.js [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
+    }
+}
+
+
+//Disables the buttons, and paints the button's ORANGE, which are in WINNING pattern.
+//It takes buttons id's as the three arguments and symbol=X or O as another argument, but the symbol īs currently not in use.
+
+//The button pressed will be disabled and symbol 'X' or 'O' will appear on it
+function setButton(button, options){
+    if(button == 1){
```
Comment (nikhilrstg18) [inline Deepak_Yadav/tic tac toe/scripts.js]: missing DRY, this function can be code with 5 to 6 lines only, rest is repeated code

File: Deepak_Yadav/tic tac toe/scripts.js (hunk 4 above)
Comment (Deepakyadav19) [reply Deepak_Yadav/tic tac toe/scripts.js]: i was optimizing this yesterday..once completely optimized i will commit changes

File: Deepak_Yadav/tic tac toe/scripts.js [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+    checker();
+}
+
+//for alternate inputs 
+function clicker(button){
+    if(options == "X"){
+        options = "O";
+        setButton(button, options);
+    }
+    else if(options == "O"){
```
Comment (nikhilrstg18) [inline Deepak_Yadav/tic tac toe/scripts.js]: do we really need else here ?

File: Deepak_Yadav/tic tac toe/scripts.js [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+        options = "X";
+        setButton(button, options);
+    }
+}
+
+function btnDisable(){
+    document.getElementById("b1").disabled = true;
```
Comment (nikhilrstg18) [inline Deepak_Yadav/tic tac toe/scripts.js]: missing DRY

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
missing DRY KISS, extra code
code optimization required
extra code
good to see folder structure and functional comments

File: Deepak_Yadav/tic tac toe/scripts.js (hunk 8 above)
Comment (Deepakyadav19) [reply Deepak_Yadav/tic tac toe/scripts.js]: will apply this before eod.
thanks nikhil for the feedback.

File: Deepak_Yadav/P1 Profile/html/about.html (hunk 2 above)
Comment (Deepakyadav19) [reply Deepak_Yadav/P1 Profile/html/about.html]: Profile page is updated as per your feedback

File: Deepak_Yadav/tic tac toe/index.html (hunk 3 above)
Comment (Deepakyadav19) [reply Deepak_Yadav/tic tac toe/index.html]: updated

File: Deepak_Yadav/P1 Profile/html/acheivement.html [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+</header>
+
+<body>
+    <section>
+        <h1 class=" text-center text-capitalize title-position mt-2"><b>acheivements</b></h1>
+        <h4 class="text-center text-danger">"<i>Failures, repeated failures, are finger posts on the road to
+                achievement.</i>"</h4>
+        <div class="container mt-5">
+            <div class="row">
+                <div class="col-lg-4 col-md-4 col-sm-6 padding-right-zerox ">
```
Comment (nikhilrstg18) [inline Deepak_Yadav/P1 Profile/html/acheivement.html]: should class be `-zerox` or `-zero` ??

**Review by nikhilrstg18 — APPROVED** [review]
looks good to me

**Review by amandeeprikhi — APPROVED** [review]
LGTM

**Comment by vishivish18** [comment]
@Deepakyadav19   Please fix the conflicts and ask for as merge

**Comment by Deepakyadav19** [comment]
> @Deepakyadav19 Please fix the conflicts and ask for as merge

Conflict resolved..Please merge this.

**Comment by Deepakyadav19** [comment]
@vishivish18 conflict resolved..

**Pull request merged**